samples/Calling/src/app/utils/AppUtils.ts

```diff
@@ -41,20 +41,12 @@ export const getGroupIdFromUrl = (): GroupLocator | undefined => {
 
 export const createGroupId = (): GroupLocator => ({ groupId: generateGUID() });
 
-/* @conditional-compile-remove(rooms) */
-export const ROOMS_API_BASE_URL = 'http://localhost:7071';
-
 /* @conditional-compile-remove(rooms) */
 export const createRoomId = async (): Promise<string> => {
```

Comment:
I think we should just rename the function to createRoom and then it just turns out we return an id to represent it has been created.

Reply:
Renamed.